disco_aws_automation/disco_elb.py

```diff
@@ -367,7 +367,7 @@ def _describe_instance_health(self, elb_id, instance_ids=None):
         return instances["InstanceStates"]
 
     def wait_for_instance_health_state(self, hostclass, testing=False, instance_ids=None, state="InService",
-                                       timeout=180):
+                                       timeout=600):
```

Comment:
This is actually waiting for instances to attach themselves to the ELB after passing smoke tests. This seems to be a fairly frequent issue that we run into and the assumption is that 3 minutes just isn't enough time for that process to complete. 10 minutes might be too much, but AWS doesn't seem to have any suggestions for how long this process should take.